pyuntl/untl_structure.py

```diff
@@ -457,6 +458,42 @@ def generate_form_data(self, **kwargs):
         # Create the form object.
         return FormGenerator(**kwargs)
 
+    def make_hidden(self):
+        """Make an unhidden record hidden."""
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                # Make the element hidden.
+                if element.content == 'False':
+                    element.content = 'True'
+                    return None
+        # Create a hidden meta record if it doesn't exist.
+        hidden_element = PYUNTL_DISPATCH['meta'](qualifier='hidden', content='True')
+        self.children.append(hidden_element)
+
+    def make_unhidden(self):
+        """Make a hidden record unhidden."""
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                # Make the element unhidden.
+                if element.content == 'True':
+                    element.content = 'False'
+                    return None
```

Comment:
Similar to comment on `make_hidden`, I think this return should be outdented a level.

Reply:
Made the necessary changes.